proto-google-cloud-gameservices-v1alpha/clirr-ignored-differences.xml

```diff
@@ -1,160 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
-<differences>
-  <difference>
-    <differenceType>7012</differenceType>
-    <className>com/google/cloud/gaming/v1alpha/*OrBuilder</className>
-    <method>* get*(*)</method>
-  </difference>
-  <difference>
-    <differenceType>7012</differenceType>
-    <className>com/google/cloud/gaming/v1alpha/*OrBuilder</className>
-    <method>boolean contains*(*)</method>
-  </difference>
-  <difference>
-    <differenceType>7012</differenceType>
-    <className>com/google/cloud/gaming/v1alpha/*OrBuilder</className>
-    <method>boolean has*(*)</method>
-  </difference>
```

Comment:
Please leave this one here. We allow adding fields to proto messages and this causes a "breaking" change in an internal interface (which should not be used by external developers)

Reply:
Done.